snapshot/bazel/ingester.go

```diff
@@ -0,0 +1,49 @@
+package bazel
+
+import (
+	"os"
+
+	log "github.com/sirupsen/logrus"
+)
+
+func (bf *bzFiler) Ingest(path string) (string, error) {
+	fileType, err := bf.getFileType(path)
+	if err != nil {
+		return "", err
+	}
+	output, err := bf.RunCmd([]string{fileType, "save", path})
+	if err != nil {
+		log.WithFields(
+			log.Fields{
+				"err":    err,
+				"output": string(output),
+				"path":   path,
+			}).Errorf("Error ingesting %s %s", fileType, path)
+		return "", err
+	}
+
+	id := string(output)
```

Comment:
Need to parse and validate output, and return ID conforming to our expectation of bz-\<digest\>-\<size\> format. In our cases, fs_util returns "\<digest\> \<size\>".

We should probably have a couple util functions for all ID-related operations - generating from hash & size and validating ID strings (similar to bazel/digest.go).